include/private/config.h.in

```diff
@@ -46,6 +46,7 @@
 
 /* function support checks */
 #cmakedefine SUPPORT_ABSTRACT_SOCKET_NAMES 1
+#cmakedefine HAVE_DISALLOW_SIGNAL_DURING_SENDING 1
```

Comment:
You'll need to add the other two `HAVE_` symbols to this file. I don't think you'll need this one anymore - you can use the `HAVE_` ones in the wrapper header instead.